docs/concepts/reask_validation.md

```diff
@@ -193,6 +193,253 @@ except ValidationError as e:
     )
 ```
 
+## Async Validation
+
+Pydantic doesn't support async validators out of the box at the moment. We've implemented two validators that you can use out of the box to run async calls.
+
+This is especially useful if you are making multiple api calls in parallel. This implementation is heavily inspired by [pydantic-async-validation](https://github.com/team23/pydantic-async-validation/tree/64e4190494141c881ade30963390f91c3286dafc).
+
+### Field Validators
+
+To use our Async Validators, you'll need to import in the `OpenAISchema` class that we expose in instructor. Here's a quick example of how we can validate multiple fields at the same time
+
+```python hl_lines="12"
+from instructor.decorators import async_field_validator
+from instructor.function_calls import OpenAISchema
```

Comment:
why not base model? we should try to not expose this externally

Reply:
would almost rather 

`from instructor import BaseModel`

Reply:
or from `instructor import AsyncBaseModel`

Reply:
Managed to get this working with the default pydantic base model! :)